From 7767097cb8563ff8f80b467f152e72270462c894 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 10 Apr 2019 21:53:12 +0100
Subject: [PATCH] OS-7725 SmartOS VMWare image creation needs to adapt to
 loader changes Reviewed by: John Levon <john.levon@joyent.com> Approved by:
 John Levon <john.levon@joyent.com>

---
 Makefile | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 57a07d6..98e8e97 100644
--- a/Makefile
+++ b/Makefile
@@ -2881,8 +2881,8 @@ SMARTOS_BITS= \
 	$(SMARTOS_BITS_DIR)/SINGLE_USER_ROOT_PASSWORD.txt \
 	$(SMARTOS_BITS_DIR)/platform-$(TIMESTAMP).tgz \
 	$(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP).iso \
-	$(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP)-USB.img.bz2 \
-	$(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP).vmwarevm.tar.bz2
+	$(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP)-USB.img.gz \
+	$(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP).vmwarevm.tar.gz
 
 .PHONY: smartos
 smartos: platform-smartos $(SMARTOS_BITS)
@@ -2891,16 +2891,16 @@ $(SMARTOS_BITS):
 	@echo "# Build smartos release: branch $(SMARTOS_LIVE_BRANCH), sha $(SMARTOS_LIVE_SHA), time `date -u +%Y%m%dT%H%M%SZ`"
 	(cd build/smartos-live \
 		&& ./tools/build_changelog \
-		&& ./tools/build_iso \
-		&& ./tools/build_usb \
-		&& ./tools/build_vmware)
+		&& ./tools/build_boot_image -I -r . \
+		&& ./tools/build_boot_image -r . \
+		&& ./tools/build_vmware -r .)
 	mkdir -p $(SMARTOS_BITS_DIR)
 	cp build/smartos-live/output/changelog.txt $(SMARTOS_BITS_DIR)
 	cp build/smartos-live/output/platform-$(TIMESTAMP)/root.password $(SMARTOS_BITS_DIR)/SINGLE_USER_ROOT_PASSWORD.txt
 	cp build/smartos-live/output/platform-$(TIMESTAMP).tgz $(SMARTOS_BITS_DIR)
 	cp build/smartos-live/output-iso/platform-$(TIMESTAMP).iso $(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP).iso
-	cp build/smartos-live/output-usb/platform-$(TIMESTAMP).usb.bz2 $(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP)-USB.img.bz2
-	cp build/smartos-live/output-vmware/smartos-$(TIMESTAMP).vmwarevm.tar.bz2 $(SMARTOS_BITS_DIR)
+	cp build/smartos-live/output-usb/platform-$(TIMESTAMP).usb.gz $(SMARTOS_BITS_DIR)/smartos-$(TIMESTAMP)-USB.img.gz
+	cp build/smartos-live/output-vmware/smartos-$(TIMESTAMP).vmwarevm.tar.gz $(SMARTOS_BITS_DIR)
 	(cd $(SMARTOS_BITS_DIR) && $(CURDIR)/tools/smartos-index $(TIMESTAMP) > index.html)
 	(cd $(SMARTOS_BITS_DIR) && /usr/bin/sum -x md5 * > md5sums.txt)
 
-- 
2.21.0

